README.md

```diff
@@ -55,14 +55,14 @@ To start OpenDevin in a docker container, run the following commands in your ter
 > When you run the following command, files in `./workspace` may be modified or deleted.
 
 ```bash
-OPENDEVIN_WORKSPACE=$(pwd)/workspace
+WORKSPACE_BASE=$(pwd)/workspace
 docker run -it \
     --pull=always \
     -e SANDBOX_USER_ID=$(id -u) \
     -e PERSIST_SANDBOX="true" \
     -e SSH_PASSWORD="make something up here" \
-    -e WORKSPACE_MOUNT_PATH=$OPENDEVIN_WORKSPACE \
-    -v $OPENDEVIN_WORKSPACE:/opt/workspace_base \
+    -e WORKSPACE_MOUNT_PATH=$WORKSPACE_BASE \
+    -v $WORKSPACE_BASE:/opt/workspace_base \
```

Comment:
Here it only change the environment vairable name. In line 58, it define the env vairable xxx, and use it in line 64, 65. The environment variable will replace as the string `$(pwd)/workspace`. So acutally it dosen't matter what this variable named. 

I didn't reproduce the issue locally. But check the issue reply [here](https://github.com/OpenDevin/OpenDevin/issues/2408#issuecomment-2164145366) and  [here](https://github.com/OpenDevin/OpenDevin/issues/2408#issuecomment-2165734666), the problem should caused by `-e PERSIST_SANDBOX="false" instead of "true"`. I know recently `PERSIST_SANDBOX` will have some problem when changing directory.

Reply:
I think opendevin want to fix [comments](https://github.com/OpenDevin/OpenDevin/issues/2408#issuecomment-2164429212) which try to rename the parameter. I think Opendevin do correctly for this proposal.